.circleci/config.yml

```diff
@@ -206,84 +216,101 @@ jobs:
             bundle add solidus --git "file://$(ruby -e"puts File.expand_path ENV['CIRCLE_WORKING_DIRECTORY']")"
             export LIB_NAME=set # dummy requireable file
             bundle exec rake -rrails -rspree/testing_support/extension_rake -e'Rake::Task["extension:test_app"].invoke'
-
-  postgres:
-    executor: postgres
+  
+  test_solidus:
+    parameters:
+      ex:
+        type: string
+        default: postgres
+      ruby_version:
+        type: string
+        default: '3.1'
+      rails_version:
+        type: string
+        default: "~> 7.0.1"
```

Comment:
nit: I think could accept just `7.0` and the `~>` be added when setting the ENV var below, this would make it look a bit more generic and also prettify the generated job name